edge-hub/core/src/Microsoft.Azure.Devices.Edge.Hub.CloudProxy/NestedDeviceScopeApiClient.cs

```diff
@@ -24,6 +24,10 @@ public class NestedDeviceScopeApiClient : IDeviceScopeApiClient
 
         const string NestedApiVersion = "2020-06-30-preview";
 
+        static readonly TimeSpan OperationTimeout = TimeSpan.FromMinutes(2);
+        // Timeout set to HttpClient is > OperationTimeout because HttpClient throws TaskCanceledException when it timeouts and retry policy won't retry it in that case
+        static readonly TimeSpan HttpOperationTimeout = TimeSpan.FromMinutes(5);
```

Comment:
Based on above comment is seems that 3 minutes would also work? So just curious on why 5? Or do we need to be greater than some multiple of operation timeout?

Reply:
it should never reach the 5 minutes timeout because it should timeout after 2 minutes, it is just as a backup in case OperationTimeout doesn't happen. It was not a specific reason I picked 5 minutes.